tests/vmi_hook_sidecar_test.go

```diff
@@ -258,3 +285,27 @@ func RenderInvalidSMBiosSidecar() map[string]string {
 		"smbios.vm.kubevirt.io/baseBoardManufacturer": "Radical Edward",
 	}
 }
+
+func RenderSidecarWithConfigMap(version string) map[string]string {
+	return map[string]string{
+		"hooks.kubevirt.io/hookSidecars": fmt.Sprintf(`[{"args": ["--version", "%s"], "image":"%s/%s:%s", "configMap": {"name": "%s","key": "%s", "hookPath": "/usr/bin/onDefineDomain"}}]`,
+			version, flags.KubeVirtUtilityRepoPrefix, sidecarShimImage, flags.KubeVirtUtilityVersionTag, configMapName, configMapKey),
+	}
+}
+
+func RenderConfigMap() *k8sv1.ConfigMap {
+	var configMapData = `#!/bin/sh
+tempFile=$(mktemp --dry-run)
+echo $4 > $tempFile
+sed -i "s|<baseBoard></baseBoard>|<baseBoard><entry name='manufacturer'>Radical Edward</entry></baseBoard>|" $tempFile
+cat $tempFile`
```

Comment:
If it isn't much trouble, would you mind using go:embed for this? You can create a testdata folder and add this script there. fca83909b8a7fc73df453443604c94b96a7b432b can be used as example.

Reply:
Thanks for the tip. I have made changes in a separate commit. Changes to `tests/BUILD.bazel` were made as a result of me running `make generate`.